twill-yarn/src/main/java/org/apache/twill/internal/appmaster/RunningContainers.java

```diff
@@ -103,6 +105,8 @@ public Integer apply(BitSet input) {
   private final Table<String, String, TwillContainerController> containers;
 
   // Map from runnableName to a BitSet, with the <instanceId> bit turned on for having an instance running.
+  private final String twillAppName;
```

Comment:
The comment above this line should be moved down to above the `runnableInstances` variable